website/markdown/posts/2020-03-16-version-1.4.0/post.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: post
+title: Tuist 1.4.0 - Lint command, more verbose logs, and configuration of the project organization
+categories: [tuist, release, swift, project generation, xcode, apple, "1.4.0"]
+excerpt: The just released version of Tuist, 1.4.0, adds support for printing more verbose logs, and configuring the Xcode organization.
+author: pepibumur
+---
+
+In difficult times where 🦠 COVID-19 presented the world with an unprecedented challenge,
+Tuist maintainers and contributors continue devoting their time to help companies scale up their Xcode projects.
+All the effort that they put into the project is reflected in a new release that we are announcing in this blog post, 1.4.0.
+Although there aren't major features shipping this time, there are small features and improvements that you might find handy for your projects.
+Let's dive right into what those improvements are.
+
+## Configure the organization name
+
+As you probably know,
+when you create a new project in Xcode, 
+one of the fields that you need to fill out is the name of the organization.
+That information ends up codified in your projects
+and Xcode uses it to create copyright headers when adding new files.
+
+Until today, 
+configuring that in Tuist was not possible,
+but thanks to [Sylvain](https://github.com/c0diq)'s [amazing contribution](https://github.com/tuist/tuist/pull/1062),
+that's finally possible.
+
+```swift
+// It can be configured at the project level
+let project = Project(organizationName: "Tuist")
+
+// Or globaly in the config file
+let config = Config(organizationName: "Tuist")
+```
+
+## Verbose argument
+
+Despite our efforts to handle all possible scenarios in Xcode projects gracefully,
+we may miss some.
+The consequence of that is that developers get unexpected results that we need to debug and fix.
+To make that work easier,
+we [introduced support for a new argument](https://github.com/tuist/tuist/pull/1027), `--verbose`,
+that configures the output to be more verbose.
+When passed to any of the commands,
+the output contains information that is useful to understand the issue and provide users with a solution as soon as possible.
+
+## Lint command
+
+As part of the project generation,
+we lint your project definition to uncover errors that otherwise would cause compilation issues later on.
+If developers wanted to run the linting logic without generating the project,
+there was no option to do so.
+Fortunately, that changes with Tuist 1.4.0 because it introduces [a new command](https://github.com/tuist/tuist/pull/1043) `tuist lint`.
+You can read more about how it works on the [commands](http://127.0.0.1:8000/docs/usage/commands/) documentation page.
```

Comment:
Does this get converted to a remote url when the markdown is converted to html?

Reply:
Ouch, not really 😅

Reply:
I pushed a commit fixing it :+1: